clickstream/src/main/kotlin/clickstream/internal/eventscheduler/CSEventScheduler.kt

```diff
@@ -168,6 +179,12 @@ internal open class CSEventScheduler(
             networkManager.eventGuidFlow.collect {
                 when (it) {
                     is CSResult.Success -> {
+                        val currentRequestIdEvents = eventRepository.getEventsOnGuId(it.value)
```

Comment:
this code will be block the `deleteEventDataByGuId` execution. can we execute the dispatch for event interceptor as a side effect?